lib/stellar_core_commander/local_process.rb

```diff
@@ -25,7 +25,8 @@ def forcescp
 
     Contract None => Any
     def initialize_history
-      run_cmd "./stellar-core", ["--newhist", "main"]
+      Dir.mkdir(history_dir) unless File.exists?(history_dir)
```

Comment:
As a aside, the "fileutils" helper that's part of the stdlib has a bunch of helpers for common file operations like this:

```ruby
require 'fileutils'

FileUtils.mkdir_p("blah")
```